2022-07_sprint/sprint_parties/media_kit.md

```diff
@@ -0,0 +1,56 @@
+# Social Media Kit
+
+(media_kit)=
+
+## Event Hashtag
+
+#DataUmbrellaPyMCSprint
+
+## Social Media Accounts
+
+| Organization   | Twitter  |  LinkedIn | Facebook | Instagram       | Discourse      |
+|----------------|----------|-----------|----------|-----------------|----------------|
+| Data Umbrella  | [@DataUmbrella](https://twitter.com/DataUmbrella) |  [dataumbrella](https://www.linkedin.com/company/dataumbrella/) | [data.umbrella.dei](https://www.facebook.com/data.umbrella.dei) | [@data.umbrella](https://www.instagram.com/data.umbrella/)  | N/A | 
+| PyMC Dev  | [@DataUmbrella](https://twitter.com/pymc_devs) |  [pymc](https://www.linkedin.com/company/pymc/mycompany/) | N/A | [pymc.io](https://discourse.pymc.io)  |
+
+
+## Share Our Events
+
+Our event series can be shared in one or more of the following ways:  
+- Event website: https://pymc-data-umbrella.xyz/en/latest/
+- Event registration form:  https://tinyurl.com/du-pymc-os-ws
+- Tweet:  https://twitter.com/DataUmbrella/status/1538875289652142080
+- LinkedIn post: ​​https://www.linkedin.com/feed/update/urn:li:activity:6944647561630662656/
```

Comment:
This one isn't recognized as a link by the linkify extension, so it needs to be hyperlinked manually:

```suggestion
- LinkedIn post: ​​[https://www.linkedin.com/feed/update/urn:li:activity:6944647561630662656/](https://www.linkedin.com/feed/update/urn:li:activity:6944647561630662656/)
```